packages/dev-server/src/main.ts

```diff
@@ -187,6 +187,7 @@ const reloadLambdas = () => {
   console.log('Change detected, rebuilding...')
   purgeRequireCache()
   lambdaFunctions = requireLambdaFunctions(PATH)
+  console.log(`rebuild completed. Listening again on ${HOSTNAME}`)
```

Comment:
I don't know if adding the hostname again is helpful. Something like a timestamp for how long it took would be interesting.